src/Propel/Generator/Platform/MysqlPlatform.php

```diff
@@ -717,10 +718,96 @@ public function getAddColumnDDL(Column $column)
         );
     }
 
+    public function getModifyTableDDL(TableDiff $tableDiff)
+    {
+        $alterTableStatements = '';
+
+        $toTable = $tableDiff->getToTable();
```

Comment:
Just got home after a couple of beers and had a chance to take a look at this. On a closer glance I think I stand by my original placement; the variable is actually used twice lower down in the function, and the place where it's used is very similar to the three places where it's used in the `DefaultPlatform` implementation. I'll still move it if you want, *but* I think if we're going to move it we should move it in `DefaultPlatform`'s `getModifyTableDDL()` too; there's no reason I can see to use a different style in the two very similar functions.